src/qt/bitcoin.cpp

```diff
@@ -282,7 +282,7 @@ void BitcoinApplication::parameterSetup()
 }
 
 void BitcoinApplication::SetPrune(bool prune, bool force) {
-     optionsModel->SetPrune(prune, force);
+    optionsModel->SetPruneTargetGB(prune ? DEFAULT_PRUNE_TARGET_GB : 0, force);
```

Comment:
In commit "gui: Fix prune in QSettings after intro" (f1ae0412d80ac7f8b30ec412564aaec782b3df0b)

I can't figure out what this commit is doing. Can the commit description be updated to say what the behavior is before and after this commit?

Reply:
See #17696.